### PR TITLE
User and Todo Crud

### PR DESCRIPTION
**Pull request by qayyum-ibrahim**
Todo and User Crud Apis

### CODE REVIEW

**Comment by qayyum-ibrahim** [comment]
Approved. Waiting for Ayoola's approval

**Review by qayyum-ibrahim — COMMENTED** [review]
Approved

**Review by CreedTech — APPROVED** [review]

**Pull request merged**
